git/cmd.py

```diff
@@ -609,6 +609,12 @@ def execute(self, command,
         # end handle
 
         try:
+            if sys.platform == 'win32':
+                CREATE_NO_WINDOW = 0x08000000
```

Comment:
`CREATE_NO_WINDOW` is used twice, is there a reason it is not a constant on module or class level ?